pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.javaslang</groupId>
+    <artifactId>javaslang-docs</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
Any ideas about versioning?

Suggestion: We could sync the version number with the code `javaslang` project, i.e. `2.0.0-SNAPSHOT`.

On the other hand the docs project will probably not be released for every minor `javaslang` version. But we should release the docs at least for every major release.